File: 10 - Python/Exercícios/04 - Vetores/Exercício 02.py
#Exercício - Soma Vetor

soma = 0

N = int(input("Quantos números você vai digitar? "))

vet = [0] * N

for i in range(N):

    vet[i] = float(input("Digite um número: "))
    soma = soma + vet[i]

media = soma / N

print("Valores = ", end="")

for i in range(N):

    print(vet[i], "|", end=" ")

print("")
print("Soma =", soma)
print("Média =", media)

File: 06 - C/Exercícios/04 - Vetores/Exercício 02.c
//Exercício - Soma Vetor

#include <stdio.h>

int main(){

    int i, N;
    double soma, media;

    soma = 0;

    printf("Quantos numeros voce vai digitar? ");
    scanf("%d", &N);

    double vet[N];

    for(i = 0; i < N; i++){

        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);

        soma = soma + vet[i];
    }

    media = soma / N;
    
    printf("\nValores = ");

    for(i = 0; i < N; i++){
        printf("%.1lf | ", vet[i]);
    }

    printf("\nSoma = %.2lf", soma);
    printf("\nMedia = %.2lf", media);

    return 0;

}